packages/FamilyTree/Reflection/FamilyRelationsAnalyzer.php

```diff
@@ -28,6 +28,10 @@ public function __construct(
      */
     public function getChildrenOfClassReflection(ClassReflection $desiredClassReflection): array
     {
+        if ($desiredClassReflection->isFinalByKeyword()) {
```

Comment:
we don't use `isFinal()` because that would also allow `@final` phpdoc, which we don't want